FND/DatesAndTimes/Occurrences.rdf

```diff
@@ -257,19 +257,20 @@ They are modularized this way to minimize the ontological committments that are
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isInputTo">
 		<rdfs:label>is input to</rdfs:label>
-		<skos:definition>indicates an input to some activity, process, system, report, analysis, etc.</skos:definition>
+		<skos:definition>indicates a precondition, requirement, or other contribution (e.g., data) to some activity, process, system, report, analysis, etc.</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isOutputFrom">
 		<rdfs:label>is output from</rdfs:label>
-		<skos:definition>indicates an output from some activity, process, system, report, analysis, etc.</skos:definition>
+		<skos:definition>indicates post condition or other product of some activity, process, system, report, analysis, etc.</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isTriggeredBy">
 		<rdfs:label>is triggered by</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-dt-oc;OccurrenceBasedDate"/>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-oc;OccurrenceKind"/>
-		<skos:definition>An OccurrenceBasedDate is triggered by an Occurrence that exemplifies the OccurrenceKind.</skos:definition>
+		<skos:definition>is activated or initiated by</skos:definition>
```

Comment:
Again imprecise, assuming that activated and initiated have different meanings. If not, and they're synonyms, just use one of them.